test/test_InfoExtractor.py

```diff
@@ -4,10 +4,13 @@
 import sys
 import unittest
 
+from yt_dlp.networking import UrllibRH
+from test.test_networking import RequestHandlerTestBase, with_request_handlers
```

Comment:
This is wrong. The yt_dlp imports should go after the path change. Basically, all test files should start like:

```py
#!/usr/bin/env python3

# Allow direct execution
import os
import sys
import unittest

sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))

import ...  # stdlib stuff

from test ...
from yt_dlp ...
```

Reply:
ah, thanks. I didn't pay much attention to that :sweat_smile: